hdfs/glob.py

```diff
@@ -0,0 +1,120 @@
+import fnmatch
+import re
+
+from hdfs import Client
+
+import posixpath
+
+
+def glob(client, hdfs_path):
+  """Return a list of paths matching a pathname pattern.
+
+  The pattern may contain simple shell-style wildcards a la
+  fnmatch. However, unlike fnmatch, filenames starting with a
+  dot are special cases that are not matched by '*' and '?'
+  patterns.
+
+  :param client: Instance of :class:`Client`.
+  :param hdfs_path: HDFS path. May contain special characters like '*', '?' and '['.
+
+    Sample usages:
+
+    .. code-block:: python
+
+      glob(client, './foo/bar/*')
+      glob(client, './foo/bar/file[0-9].txt')
+      glob(client, './foo/bar/file?.txt')
+
+  """
+  return list(iglob(client, hdfs_path))
```

Comment:
`glob` provides little value over `iglob`: it saves just the `list` call. What do you think about only exposing `iglob`, possibly renaming it to `glob`?